### DISCUSSION TITLE
Can i use netinet/tcp.h file for code?

### DISCUSSION THREAD

**Discussion by xhaktm3** [ANLAB-KAIST/KENSv3#207] [Project Q&A 4. Etc] [answered]
I already submit PA2 file that coded with netinet/tcp.h file's tcp_info or tcpheader structure. I coded it in linux OS, but i start PA3 in MacOS. In MacOS, it has different netinet/tcp.h file from linux which has the structure name "tcp_connection info" not "tcp_info". So, i wanna ask to TA about which os PAs use for building and testing student's code? Do i need to specify the os that i use in code?

**Comment by Pusnow** [comment, marked as answer]
Linux. Please test your source code is compatible with Linux before submission.
